CONTRIBUTING.md

```diff
@@ -42,13 +42,15 @@ If applied, this commit will _Your subject line here_
 For the submission of more substantial changes or additions, an issue should be opened outlining what is being proposed for implementation. The title of the issue should be descriptive and brief, follow the same rules as a commit message, as outlined above. The body of the issue should detail the reasoning for the need of the work to be carried out and the desired outcome.
 
 
-## Code Formatting
+## Code Formatting and Commentary
 
 ### Javascript
 All Javascript must be formatted with [ESLint](http://eslint.org/) using the JAAK [configuration](https://github.com/jaakmusic/eslint-config-jaak).
 
 ### Go
 Go code should match the output of `gofmt -s`.
 
+Go code must be documented adhering to the official Go [commentary](https://golang.org/doc/effective_go.html#commentary) guidelines.
```

Comment:
Let's say "Go code should" rather than "must" as we don't *require* this in order to merge code (e.g. a critical bug fix), but do recommend it.